package.json

```diff
@@ -41,6 +41,7 @@
     "electron-store": "5.1.1",
     "hex-string": "1.0.3",
     "history": "4.10.1",
+    "html-react-parser": "1.2.6",
```

Comment:
We don't need this dependency since instead of HTML, we can pass JSX

Reply:
It has been updated. Please check